minkindr/include/kindr/minimal/implementation/angle-axis-inl.h

```diff
@@ -64,12 +64,15 @@ AngleAxisTemplate<Scalar>::AngleAxisTemplate(const RotationMatrix& matrix) :
 
 template<typename Scalar>
 AngleAxisTemplate<Scalar>::AngleAxisTemplate(const Vector3& angleAxis) {
```

Comment:
Could you maybe rename this parameter to `rotationVector` for correctness and consistency?